modules/users/server/jobs/user-finish-signup.server.job.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+/**
+ * Task that checks for users who didn't finish their signup process and thus
+ * have `public:false` in their profile. The script sends 3 reminder emails
+ * to these users in 3 day intervals, starting 24h after signup.
+ *
+ * Keeps count of reminder emails at user's model.
+ */
+
+
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+    emailService = require(path.resolve('./modules/core/server/services/email.server.service')),
+    async = require('async'),
+    moment = require('moment'),
+    mongoose = require('mongoose'),
+    User = mongoose.model('User');
+
+// How many reminder emails we should send?
+var maxReminders = 3;
+
+// How long time between signup and first reminder?
+var firstReminderAfter = { 'hours': 4 }; // days|hours|minutes
+
+// How long time between reminders?
+var nextReminderAfter = { 'days': 2 }; // days|hours|minutes
+
+module.exports = function(job, agendaDone) {
+  async.waterfall([
+
+    // Find un-confirmed users
+    function(done) {
+
+      // Ignore very recent messages and look for only older than x seconds
+      var createdTimeAgo = new Date();
+      createdTimeAgo.setSeconds(createdTimeAgo.getSeconds() - moment.duration(firstReminderAfter).asSeconds());
```

Comment:
seeing as `moment` is available, could be something like `moment().subtract(firstReminderAfter).toDate()` (didn't run it)